src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -73,12 +74,11 @@
                         (map #((keyword %) data)
                              (concat [:library:indication :library:dataUseRestriction :library:numSubjects]
                                      extra-columns))))}]))
-   :component-will-receive-props
-   (fn [{:keys [next-props refs]}]
-     (let [current-search-text (:filter-text (react/call :get-query-params (@refs "table")))
-           new-search-text (:search-text next-props)]
-       (when-not (= current-search-text new-search-text)
-         (react/call :update-query-params (@refs "table") {:filter-text new-search-text}))))
+   :execute-search
+   (fn [{:keys [refs]}]
+     (when (not (nil? (@refs "table")))
+       (react/call :update-query-params (@refs "table") {:current-page 1})
+       (react/call :execute-search (@refs "table"))))
```

Comment:
indentation - shouldn't this be:
```
     (when (not (nil? (@refs "table")))
       (react/call :update-query-params (@refs "table") {:current-page 1})
     (react/call :execute-search (@refs "table"))))
```
I find this pretty important, because the `:execute-search` should not look like it is conditional on the `when`